hack/prerelease.sh

```diff
@@ -0,0 +1,136 @@
+#!/usr/bin/env bash
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+SCRIPT_DIR="$(
+    cd "$(dirname "$0")" >/dev/null
+    pwd
+)"
+
+PROJECT_DIR="$(
+    cd "$SCRIPT_DIR/.." >/dev/null
+    pwd
+)"
+
+usage() {
+    echo "
+Usage:
+    ${0##*/} [options]
+
+Optional arguments:
+    -p, --product $PRODUCT
+        The product on which to activate the pre-release subscription.
+        Can be specified multiple times.
+    -d, --debug
+        Activate tracing/debug mode.
+    -h, --help
+        Display this message.
+
+Example:
+    ${0##*/}
+" >&2
+}
+
+parse_args() {
+    PRODUCT_LIST=()
+    while [[ $# -gt 0 ]]; do
+        case $1 in
+        -p|--product)
+            case $2 in
+            developerHub|dh|rhdh)
+                PRODUCT_LIST+=( "rhdh" )
+                ;;
+            gitops|pipelines)
+                PRODUCT_LIST+=( "$2" )
+                ;;
+            *)
+                echo "[ERROR] Unknown product: $1"
+                usage
```

Comment:
when the product is invalid, it should exist the script. So here it's missing  `exit 1`

Reply:
Done